src/main/java/com/assignment1/item/ItemEntity.java

```diff
@@ -0,0 +1,94 @@
+package com.assignment1.item;
+
+import com.assignment1.Type;
+
+/** Class as blueprint for Item Entity. */
+@SuppressWarnings("PMD.DataClass")
+public class ItemEntity {
+
+  /** Name of item. */
+  private String name;
+  /** type(RAW, MANUFACTURED, IMPORTED) of item. */
+  private Type type;
+  /** quantity of item. */
+  private int quantity;
+  /** price of item. */
+  private double price;
+  /** tax of item. */
+  private double tax;
```

Comment:
try to use `BigDecimal` instead when dealing with money

Reply:
Used BitDecimal for price in the changed code